spec/controllers/proposals_controller_spec.rb

```diff
@@ -173,6 +173,24 @@
           expect(query[2].id).to be(single.id)
         end
       end
+
+      it "returns JSON for preview count" do
+        login_as(user)
+        proposals = 3.times.map do |i|
+          wo = create(:test_client_request, project_title: "Work Order #{i}")
+          wo.proposal.update(requester: user)
+          wo.proposal.reindex
```

Comment:
do we need to call `reindex` here if we are refreshing entire index on L 185 anyway?